velox/vector/tests/SelectivityVectorTest.cpp

```diff
@@ -360,6 +360,47 @@ TEST(SelectivityVectorTest, iterator) {
   EXPECT_EQ(count, bits::countBits(&contiguous[0], 0, 240));
 }
 
+TEST(SelectivityVectorTest, selectivityIteratorCopyConstructor) {
+  SelectivityVector vector(10);
+  vector.clearAll();
+  vector.setValid(1, true);
+  vector.setValid(3, true);
+  vector.setValid(5, true);
+  vector.setValid(7, true);
+  vector.updateBounds();
+
+  // Test non-advanced iterator
+  SelectivityIterator iter1(vector);
+  SelectivityIterator iter1Copy(iter1);
+
+  auto collectValues = [](SelectivityIterator& iter) {
+    std::vector<vector_size_t> values;
+    vector_size_t row;
+    while (iter.next(row)) {
+      values.push_back(row);
+    }
+    return values;
+  };
+
+  EXPECT_EQ(collectValues(iter1), std::vector<vector_size_t>({1, 3, 5, 7}));
```

Comment:
could we just compare the iterators directly, if you provide the equality operator somewhere?

Reply:
I am not sure I know how to do that. I thought comparing iterators directly would result in comparing the element they point to which is insufficient here. 
I could be wrong though - do you have a suggestion how to do that?